runner_service/utils.py

```diff
@@ -206,18 +207,21 @@ def timeout_handler():
             raise SSHTimeout
 
         socket.setdefaulttimeout(self.timeout)
-        s = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
         try:
-            s.connect((self.host, self.port))
+            family, type, proto, canonname, sockaddr = socket.getaddrinfo(self.host, self.port, 0, 0, socket.SOL_TCP)[0]
```

Comment:
`type, proto, canonname,` - we don't actually use any of these, so it's better to replace them with `_, _, _,` - it's a convention.
Also, since we're explicitly asking for a `SOCK_STREAM` type of socket it would be nice  to pass that to the `type` argument.